draft-ietf-rtcweb-jsep.xml

```diff
@@ -1996,13 +1998,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <list style="symbols">
 
               <t>An "a=mid" line, as specified in
-              <xref target="RFC5888"></xref>, Section 4. All mid values
+              <xref target="RFC5888"></xref>, Section 4. All MID values
               MUST be generated in a fashion that does not leak user
               information, e.g., randomly or using a per-PeerConnection counter, and
               SHOULD be 3 bytes or less, to allow them to efficiently fit into
               the RTP header extension defined in
               <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation">
-              </xref>, Section 14.</t>
+              </xref>, Section 14. Note that the mid attribute of the
```

Comment:
Not sure we need to go into this level of detail here in this bullet list. I would just say

> Note that this does not set the RtpTransceiver mid attribute, as that only occurs when the description is applied. The generated MID value can be considered a "proposed" MID at this point.